src/machine/behavior/is/bitwise/and.rs

```diff
@@ -1,5 +1,6 @@
 use machine::state::State;
 
+/// bitwise and
```

Comment:
I think, the doc-comments should be a little bit more detailed. But I suggest to do this in the end, when we have finished the other task.